app/controllers/users/confirmations_controller.rb

```diff
@@ -6,9 +6,20 @@ def show
     self.resource = resource_class.confirm_by_token(params[:confirmation_token])
 
     if resource.errors.empty?
+      notify_school_primary_contact
```

Comment:
We only want to do this for school induction coordinators, right?

Reply:
Yes. I agree, best to check that here